### PR TITLE
analog: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**
GetExAbsArchiveDaysMC, GetExAbsArchiveHoursMC

### CODE REVIEW

File: docs/analog/commands/GetArchiveHoursMC.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ It`s full value of pulse counter with diff for each previous hours (8 hours if r
 | command id   | `26`                      |                      | `0x1a`   |
 | command size | `4`                       |                      | `0x04`   |
 | start date   | `2023.12.23 00:00:00 GMT` | `0b0010111110010111` | `0x2f97` |
-| hours        | hour: `12:00`, hours: `2` | `0b01001100`         | `0x4c`   |
+| hours        | hour: `12:00`, hours: `2` | `0b01001100`         | `0x2c`   |
```
Comment (DarkPark) [inline docs/analog/commands/GetArchiveHoursMC.md:47]: but `0b01001100` is `4c`

File: docs/analog/commands/GetExAbsArchiveHoursMC.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+#### `1` channel:
+
+| Field        | Value                     | Bits                 | Hex      |
+| ------------ | ------------------------- | -------------------- | -------- |
+| extra flag   | `31`                      |                      | `0x1f`   |
+| command id   | `12`                      |                      | `0x0c`   |
+| command size | `4`                       |                      | `0x04`   |
+| start date   | `2023.12.23 00:00:00 GMT` | `0b0010111110010111` | `0x2f97` |
+| hours        | hour: `12:00`, hours: `2` | `0b01001100`         | `0x2c`   |
```
Comment (DarkPark) [inline docs/analog/commands/GetExAbsArchiveHoursMC.md:49]: here as well

File: docs/analog/commands/GetArchiveHoursMC.md (hunk 1 above)
Comment (DeadByElpy) [reply docs/analog/commands/GetArchiveHoursMC.md]: fixed

File: docs/analog/commands/GetExAbsArchiveHoursMC.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/analog/commands/GetExAbsArchiveHoursMC.md]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**